### PR TITLE
add html, javascript modules, js file and css

### PR DESCRIPTION
**Pull request by businesskaniki**
i added  four files in js module folder  and a html fille and also a css file

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  static remove(bookId) {
+    const currentBooks = JSON.parse(localStorage.getItem('mybooks'));
+    const leftOver = currentBooks.filter(
+      (book) => Number(book.id) !== Number(bookId),
+    );
+
+    localStorage.setItem('mybooks', JSON.stringify(leftOver));
+  }
+}
```
Comment (thecodechaser) [inline modules/books.js]: - [ ] Please change all regular functions to arrow functions in this class to follow ES6 syntax.

File: modules/nav.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        pages.forEach((page) => page.classList.add('disable-page'));
+        currentPage.classList.remove('disable-page');
+        currentPage.classList.add('showing');
+      }
+    };
+
+    nav.addEventListener('click', (e) => showPage(e));
+  }
+}
+
```
Comment (thecodechaser) [inline modules/nav.js]: - [ ] Please change all regular functions to arrow functions in this class to follow ES6 syntax.

File: modules/userinterface.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        luxon.DateTime.DATE_FULL,
+      )} ${dt.toLocaleString(luxon.DateTime.TIME_WITH_SECONDS)}`;
+      clock.innerText = theTime;
+    };
+
+    setInterval(() => {
+      showTime();
+    });
+  }
+}
```
Comment (thecodechaser) [inline modules/userinterface.js]: - [ ] Please change all regular functions to arrow functions in this class to follow ES6 syntax.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @businesskaniki👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @businesskaniki , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 
- Clean code :heavy_check_mark:  
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**